cleanlab_studio/studio/studio.py

```diff
@@ -166,15 +166,15 @@ def apply_corrections(self, cleanset_id: str, dataset: Any, keep_excluded: bool
                 .drop("action")
             )
         elif isinstance(dataset, pd.DataFrame):
-            cl_cols = self.download_cleanlab_columns(cleanset_id)
+            cl_cols = self.download_cleanlab_columns(cleanset_id, include_project_details=True)
             joined_ds: pd.DataFrame
             if id_col in dataset.columns:
                 joined_ds = dataset.join(cl_cols.set_index(id_col), on=id_col)
             else:
                 joined_ds = dataset.join(cl_cols.set_index(id_col).sort_values(by=id_col))
-            joined_ds["__cleanlab_final_label"] = joined_ds["clean_label"].where(
-                np.asarray(list(map(check_not_none, joined_ds["clean_label"].to_numpy()))),
-                dataset[label_column].to_numpy(),
+            joined_ds["__cleanlab_final_label"] = joined_ds["corrected_label"].where(
+                joined_ds["corrected_label"].apply(check_not_none).tolist(),
+                dataset[label_column].tolist(),
```

Comment:
Can you try to check if we need to do `.apply(check_not_none)`? We might be able to just use pandas `.isna()` since we [convert](https://github.com/cleanlab/cleanlab-studio-backend/blob/27ddbf189c8cad66def1fa371caf2a34c89dd997/backend/utils/export/export_util.py#L204) columns to dtypes where null values are represented by [`pd.NA`](https://pandas.pydata.org/docs/user_guide/missing_data.html#experimental-na-scalar-to-denote-missing-values) in the backend. Since we use json to encode the result in the api response, we shouldn't lose any information about null values. 

Let me know if you need any guidance on how to verify this. Would be good to check for both pyspark and pandas cases (I can get you a databricks account so you can test the pyspark version).